LCARSToolkit/LCARSToolkit.Controls/Button.cs

```diff
@@ -19,6 +19,28 @@ namespace LCARSToolkit.Controls
 {
     public class Button : Windows.UI.Xaml.Controls.Button
     {
+
+        public Button()
+        {
+            this.DefaultStyleKey = typeof(Button);
+
+            this.SizeChanged += (s,e) => UpdateCorners();
+            this.Click += (s,e) => SoundElement?.Play();
+            // subscribe to the single, global FlashTimer such that everything flashes synchronously
+            Extensions.FlashTimer.Tick += FlashTimer_Tick;
+            this.Unloaded += Button_Unloaded;
+        }
+
+        private void FlashTimer_Tick(object sender, object e)
+        {
+            IsLit = (Illumination == Illumination.Flashing) ? !IsLit : (Illumination == Illumination.On);
+        }
+
+        private void Button_Unloaded(object sender, RoutedEventArgs e)
+        {
+            Extensions.FlashTimer.Tick -= FlashTimer_Tick;
```

Comment:
Is it necessary to detach the `Button_Unloaded` handler too?

Reply:
It should be enough to cleanup references to this object on other objects (in this case the reference of the button on the static flashtimer)